### PR TITLE
Add environment variables for running test suite.

### PR DESCRIPTION
**Pull request by mohit-rocks**
Add base environment variables file.
Update setup file to copy it at appropriate place.

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for all features, enhancements and bug fixes (until 3.3.0 is released) 
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no (Not required)
| Related user documentation PR URL      | N/A
| Related developer documentation PR URL | N/A
| Issue(s) addressed                     | Fixes #... 

#### Description:
Adding sample environment variables that can be picked up while running PHPUnit test cases on local.

#### Steps to test this PR:
1. Pull the PR on local. We can't test it on MautiBox as this is related to local setup.
2. Run ` sh .ddev/mautic-setup.sh`
3. When you run above script, ensure that ddev setup is not pre-configured. 
4. This will copy `.env.dist` from `.ddev` folder to the main folder of the project.

### CODE REVIEW

**Comment by mohit-rocks** [comment]
Related original PR #9669

File: .gitignore [hunk 1]
```diff
@@ -4,6 +4,7 @@
 !.htaccess
 !.gitkeep
 !.ddev
+!.ddev/.env.dist
```
Comment (dennisameling) [inline .gitignore]: This line is superfluous 😊 it'll also work without this line 👍🏼 

```suggestion
```

File: .ddev/.env.dist [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# .env.test
```
Comment (dennisameling) [inline .ddev/.env.dist]: We can remove this comment 😊 

```suggestion
```

**Review by dennisameling — CHANGES_REQUESTED** [review]
This looks great, just two minor nits! 🚀

**Review by dennisameling — CHANGES_REQUESTED** [review]
Sorry but you removed the line in the wrong `.env.dist` file, please see the "Files changed" tab 😊  I requested the change on the `.ddev/.env.dist`. Thanks in advance 🚀

**Comment by mohit-rocks** [comment]
Hi @dennisameling 
Sorry for the back n forth. I've updated the appropriate files. 
Please review the changes.

**Review by dennisameling — DISMISSED** [review]
Awesome! Thank you! 🚀

**Comment by npracht** [comment]
We can add that to 3.3.2 right ? I can label it `chore` also right ?

**Comment by dennisameling** [comment]
Let's keep it as a bug, since it affects developers who are cloning Mautic from GitHub and using DDEV to contribute to it. 3.3.2 is fine 👍🏼

**Comment by RCheesley** [comment]
Changed the base to 3.3 for merging in 3.3.2 - if all tests pass we should be good to go but it may well need to be rebased @mohit-rocks !

**Comment by RCheesley** [comment]
@mohit-rocks if you have a moment can you rebase this for the features branch if we are to merge it in Mautic 4? Thanks!

**Comment by RCheesley** [comment]
As this has not yet been rebased we will have to get this into a later release.

**Pull request merged**